test/functorch/test_aotdispatch.py

```diff
@@ -2613,7 +2612,16 @@ def f(args):
             return op.op(*c_args, **c_kwargs)
 
         compiled_f = compiled_function(f, nop, nop)
-        _test_aot_autograd_forwards_backwards_helper(self, f, compiled_f, args)
+        try:
+            _test_aot_autograd_forwards_backwards_helper(self, f, compiled_f, args)
+        except GuardOnDataDependentSymNode:
+            # Carveout for getitem; I don't want to xfail the entire test
+            # because that will reject known to be good tests see
+            # https://github.com/pytorch/pytorch/issues/94705
+            if op.name == "__getitem__":
+                self.skipTest("Dynamic output shape operation in trace")
```

Comment:
Minor nit - which tests is this? Should we limit it to a list?

Reply:
It's *one* of the sample inputs for getitem. The sample inputs are not named so there is no way to selectively select them.